partner_identifiers/README.rst

```diff
@@ -0,0 +1,101 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==============================
+Partner Identification Numbers
+==============================
+
+This module allows to manage all sort of identification numbers
+and certificates which are assigned to a partner and vary from country
+to country.
+
+* Commercial register
+* VAT ID (eventually)
+* Fiscal ID's
+* Membership numbers
+* Driver license
+* ...
+
+
+Installation
+============
+
+Install this addon from Settings > Local Modules, as usual.
+
+
+Configuration
+=============
+
+Configure all ID types you need in Sales > Configuration > Address Book > Partner ID Categories.
+For example, we create a category 'Driver License':
+
+* Name : Name of this ID type. For example, 'Driver License'
+* Code : Code, abbreviation or acronym of this ID type. For example, 'driver_license'
+* Python validation code: Optional python code called to validate ID numbers of this ID type.
+
+Usage
+=====
+
+In partner form you will see another tab called 'ID Numbers'. You can add
+any IDs to this partner, defining:
+
+* Category    : ID type defined in configuration. For example, Driver License
+* ID Number   : The ID itself. For example, Driver License number of this person
+* Issued by   : Another partner, who issued this ID. For example, Traffic National Institution
+* Valid from  : Issued date. For example, date when person approved his driving exam, 21/10/2009
```

Comment:
I would add the following, at least necessary for passports and visa:
* Place of Issuance